### PR TITLE
Add service-info endpoint, fix #27

### LINKED ISSUE #27: Add service-info endpoint
Now that [Service Info](https://github.com/ga4gh-discovery/ga4gh-service-info/releases/tag/v1.0.0) has been approved and released, we should add the corresponding endpoint.

### PR DESCRIPTION
**Pull request by mcupak**

### CODE REVIEW

**Review by jfuerth — APPROVED** [review]
Looks good to me. The tags are a good idea to show how the endpoints map to different sections of the spec.

File: spec/search-api.yaml [hunk 1]
```diff
@@ -17,6 +17,8 @@ paths:
       summary: List Tables
       description: Returns a list of Tables.
       operationId: listTables
+      tags:
+        - Tables API
```
Comment (kemp-google) [inline spec/search-api.yaml]: Are these intended to be standardized in any way?

The service-info API seems to use 'service-info' to describe itself...

File: spec/search-api.yaml [hunk 2]
```diff
@@ -88,6 +96,24 @@ paths:
           description: This server does not implement the search operation
         '500':
           description: An unexpected error occurred
+  /service-info:
+    get:
+      summary: 'Show information about the Search service'
+      operationId: getServiceInfo
+      tags:
+        - Service Info API
```
Comment (kemp-google) [inline spec/search-api.yaml]: As noted above the service info API appears to use the tag 'service-info' ... I would think we should standardize these in some way but failing that we should probably at least be consistent.

**Comment by mcupak** [comment]
@kemp-google Like this?

**Review by FizzDNA — COMMENTED** [review]
Echoing @jfuerth 's comments, these changes look good to me. I only have one question: does adding a new endpoint not warrant a version bump?

**Comment by mcupak** [comment]
Thanks! That's a good point, I'll bump it to 0.2 just so that we don't conflict with the previously released 0.1.

**Comment by jfuerth** [comment]
LGTM

**Pull request merged**